java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -279,4 +280,38 @@ CompletableFuture<Double> zaddIncr(
      * }</pre>
      */
     CompletableFuture<Double> zscore(String key, String member);
+
+    /**
+     * Returns the difference between the first sorted set and all the successive sorted sets.
+     *
+     * @see <a href="https://redis.io/commands/zdiff/">redis.io</a> for more details.
+     * @param keys The keys of the sorted sets.
+     * @return An <code>array</code> of <code>elements</code> representing the difference between the
```

Comment:
Looks like the existing javadocs are inconsistent on whether they wrap "elements" in a code block. They mostly do not but there are a couple instances where they do. Since most don't, should we remove the code block here? Either way, the @return for zdiffWithScores does not wrap "elements" but this one does, so we should change one of them so that they are consistent

Reply:
it probably shouldn't wrap them if it is not a param I will revert.